docs/en/ingest-arch/13-multitenant.asciidoc

```diff
@@ -16,7 +16,31 @@ These logs need to be sent to different {es} clusters and to S3 for backup and a
 In such use cases, agents send the data to {ls} as a routing mechanism to different destinations. 
 Note that the System and Windows integrations must be installed on all {es} clusters to which the data is routed.
 
-// ToDo: Add config example/logic in Logstash for routing data to different destinations
+Sample config::
++ 
+[source,ruby]
+input {
+  elastic_agent {
+    port => 5044
+  }
+}
+output {
```

Comment:
@jsvd I'd like to hold the config to its most basic elements for clarity.  Is there's a filter (or filters) that would most likely be needed for this example to make sense (mutate maybe)? If so, I'll add it.